site/content/docs/developer-guide/plugin-manifest.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Writing Krew plugin manifests
+slug: plugin-manifest
+weight: 100
+---
+
+Each Krew plugin has a "plugin manifest", which is a YAML file that describes
+the plugin, how it can be downloaded and installed on a machine.
+
+It is **highly recommended** you look at [example plugin manifests]({{< ref
+"example-plugin-manifests.md" >}}) to copy from an existing plugin and adapt to
+your needs instead of learning everything in this page.
+
+#### Sample plugin manifest
+
+Here's a sample manifest file that installs a bash-based plugin that supports
+only Linux and macOS:
+
+```yaml
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest.
+  name: restart
+spec:
+  # 'version' is a valid semantic version string (see semver.org)
+  version: "v0.0.1"
+  # 'homepage' usually links to the GitHub repository of the plugin
+  homepage: https://github.com/achanda/kubectl-restart
+  # 'shortDescription' explains what the plugin does in only a few words
+  shortDescription: "Restarts a pod with the given name"
+  description: |
+    Restarts a pod with the given name. The existing pod
+    will be deleted and created again, not a true restart.
+  # 'platforms' specify installation methods for various platforms (os/arch)
+  platforms:
+  - selector:
+      matchExpressions:
+      - key: "os"
+        operator: "In"
+        values:
+        - darwin
+        - linux
+    # 'uri' specifies .zip or .tar.gz archive URL of a plugin
+    uri: https://github.com/achanda/kubectl-restart/archive/v0.0.3.zip
+    # 'sha256' is the sha256sum of the url above
+    sha256: d7079b79bf4e10e55ded435a2e862efe310e019b6c306a8ff04191238ef4b2b4
+    # 'files' lists which files should be extracted out from downloaded archive
+    files:
+    - from: "kubectl-restart-*/restart.sh"
+      to: "./restart.sh"
+    - from: "LICENSE"
```

Comment:
It doesn't say anywhere that we require a license to be included and installed. Let's add something like
```yaml
# make sure to always install your LICENSE file
```

**EDIT** Ah, I found that you added a not to the submitters checklist. So this comment is not super urgent, but maybe still helpful.

Reply:
Wontfix for now.